cli/src/workers.ts

```diff
@@ -1,266 +0,0 @@
-}: BuildWatcherOptions<T>): { watcher: chokidar.FSWatcher } {
-  const watcher = chokidar.watch(included, { ignored, cwd: process.cwd() });
-
-  let stopServer: () => Promise<void> | undefined;
-  const modules: Record<string, T[]> = {};
-
-  const updateModule = async (path: string) => {
-    modules[path] = await compile(path);
-    if (run) {
-      if (stopServer) await stopServer();
-      stopServer = await run(Object.values(modules).flat());
-    }
-  };
-
-  watcher
-    .on('add', async (path) => {
-      console.log(`[watcher] add ${path}`);
-      modules[path] = [];
-    })
-    .on('change', async (path) => {
-      console.log(`[watcher] change ${path}`);
-      await updateModule(path);
-    })
-    .on('error', (error) => {
-      console.error(`[watcher] error ${error}`);
-      throw new Error('Watcher error');
-    })
-    .on('ready', async () => {
-      console.log('[watcher] ready');
-
-      for await (const path of Object.keys(modules)) {
-        modules[path] = await compile(path);
-      }
-
-      if (run) {
-        stopServer = await run(Object.values(modules).flat());
-      }
-    });
-
-  return { watcher };
-}
-
-export async function compileWorkers(file: string): Promise<WorkerScript[]> {
-  const external: string[] = [];
-  const functions: Record<string, string> = {};
-
-  try {
-    babel.transformFileSync(file, {
-      presets: [presetTypeScript, presetReact],
-      plugins: [
-        (): PluginItem => {
-          return {
-            visitor: {
-              ImportDeclaration: {
-                enter(path) {
-                  for (const specifier of path.node.specifiers) {
-                    const binding = path.scope.getBinding(specifier.local.name);
-                    if (!binding) continue;
-                    const refPaths = binding.referencePaths;
-                    for (const refPath of refPaths) {
-                      const usedInWorker = !!refPath.find((path) => {
-                        if (!path.isFunction()) return false;
-                        return isXataWorker(path);
-                      });
-
-                      if (usedInWorker) {
-                        external.push(path.toString());
-                      }
-                    }
-                  }
-                }
-              },
-              VariableDeclaration: {
-                enter() {
-                  // external.push(path.toString());
-                }
-              },
-              Function: {
-                enter(path) {
-                  if (isXataWorker(path)) {
-                    const args = (path.parent as CallExpression).arguments as any[];
-                    const workerName = args[0]?.value;
-                    if (!workerName || typeof workerName !== 'string') {
-                      console.error(`Found a worker without a name in file ${file}`);
-                    } else {
-                      functions[workerName] = path.toString();
```

Comment:
Is `path` here the path to he user defined worker module which could be defined anywhere in their project, which will have its contents transpiled, bundled below, wrapped in this `workerCode` and then uploaded to CF? 

The user was able to define their own handlers that would be executed in CloudFlare at a certain endpoint right?